Eastern-Research-Group/csb-rebate-forms-app
# Feature/update nces fetching

## Related Issues:
* CSBAPP-311

## Main Changes:
* Update to #394 – Updates server to fetch NCES.json data before starting Express app, and stores it on the Express app's locals object for use in the Formio NCES endpoint.

## Steps To Test:
1. Hit the app's NCES endpoint several times (replicating several concurrent users hitting that endpoint) and ensure the memory use doesn't spike causing the app to crash.
